spec/spec_helper.rb

```diff
@@ -1,7 +1,6 @@
 require_relative 'support/simplecov' if ENV['COVERAGE'] == 'true'
 
 require 'pry'
-require_relative '../ext/RMagick/extconf'
```

Comment:
@Watson1978 I added this in order to generate test coverage for the `extconf.rb` file. I guess we could only load it when `ENV['COVERAGE'] == 'true'`.